File: AMF/ClientApp/src/app/dtos/Subscribers.ts
export class Subscribers {
  subscriptionID: String;
  firstName: String;
  lastName: String;
  gender: String;
  birthDate: Date;
  countryID: String;
  emailID: String;
  city: String;

  new() {
    this.subscriptionID = null;
    this.firstName = null;
    this.gender = null;
    this.birthDate = null;
    this.countryID = null;
    this.emailID = null;
    this.city = null;
  }
}

File: AMF.Entities/DTOs/Subscribers.cs
using System;

using System.ComponentModel.DataAnnotations;

namespace AMF.Entities.DTOs
{
    public class Subscribers
    {
        [Key]
        public Guid SubscriptionID { get; set; }
        public String FirstName { get; set; }
        public String LastName { get; set; }
        public String Gender { get; set; }
        public DateTime BirthDate { get; set; }
        public Guid CountryID { get; set; }
        public Guid EmailID { get; set; }
        public String City { get; set; }
        public String PostCode { get; set; }

    }
}
